### ISSUE TITLE
Tab Component freezes Application and causes error 3221226525

### ISSUE THREAD

**Issue by Skayo** [yue/yue#89]
I have this simple code:
```javascript
const win = gui.Window.create({})
win.setContentSize({width: 400, height: 400})
win.setTitle('Yue Test')
win.onClose = () => gui.MessageLoop.quit()

const tabs = gui.Tab.create();
win.setContentView(tabs);

const testTab = gui.Container.create();
tabs.addPage('Test', testTab);

const testText = gui.Label.create('test');
testTab.addChildView(testText);

const testTab2 = gui.Container.create();
tabs.addPage('Test2', testTab2);

const testText2 = gui.Label.create('test2');
testTab.addChildView(testText2);
```

But when I start the app and try to change tabs, the application suddenly freezes and stops.
In the console it outputs an error 3221226525 (ELIFECYCLE)

No clue what I'm doing wrong...

OS: Windows 10 (64 bit)
Yue: 0.6.3
Node.js: v10.15.3
Yode: v0.4.2

**Comment by zaaack** [comment]
this might be relavant to #88

**Comment by zcbenz** [comment]
Fixed with c868f1f.
